src/Tasks/Microsoft.NET.Build.Tasks/DependencyContextBuilder.cs

```diff
@@ -185,6 +193,40 @@ private static string GenerateRuntimeSignature(IEnumerable<LockFileTargetLibrary
             return dependencies;
         }
 
+        private RuntimeLibrary CreateRuntimeLibrary(string type,
```

Comment:
The formatting style is varying here. Please, when you put an argument on a newline, put them all in a new line. So, string type should be on the line below.

Also, the closing parenthesis should be in the same line as string hashPath = null